src/Symfony/Component/Translation/Translator.php

```diff
@@ -444,6 +430,14 @@ private function getFallbackContent(MessageCatalogue $catalogue)
             $fallbackSuffix = ucfirst(preg_replace($replacementPattern, '_', $fallback));
             $currentSuffix = ucfirst(preg_replace($replacementPattern, '_', $current));
 
+            $fallbackMessages = $fallbackCatalogue->all();
```

Comment:
This should be in `else` to avoid call if not needed as you replace it if not in debug anyway...